builtin/locale/__builtin.ru.tr

```diff
@@ -0,0 +1,246 @@
+Reverting actions of player '@1' since @2 seconds.=Отмена действия игрока "@1" за последние @2 секунд(у/ы).
+Reverting actions of @1 since @2 seconds.=Отмена действий @1 за последние @2 секунд(у/ы).
+(log is too long to show)=(журнал слишком длинный для отображения)
+Reverting actions succeeded.=Отмена действий завершилась успешно.
+Reverting actions FAILED.=Отменить действия НЕ УДАЛОСЬ.
+Show server status=Показать статус сервера
+This command was disabled by a mod or game.=Эта команда была отключена модом или игрой.
+[<0..23>:<0..59> | <0..24000>]=[<0..23>:<0..59> | <0..24000>]
+Show or set time of day=Показать или установить время суток
+Current time is @1:@2.=Текущее время @1:@2 .
+You don't have permission to run this command (missing privilege: @1).=У вас нет разрешения на запуск этой команды (отсутствует привилегия: @1).
+Invalid time (must be between 0 and 24000).=Недопустимое время (должно быть в диапазоне от 0 до 24000).
+Time of day changed.=Время суток изменено.
+Invalid hour (must be between 0 and 23 inclusive).=Недопустимый час (должно быть от 0 до 23 включительно).
+Invalid minute (must be between 0 and 59 inclusive).=Недопустимая минута (должно быть от 0 до 59 включительно).
+Show day count since world creation=Показать количество дней с момента сотворения мира
+Current day is @1.=Текущий день: @1.
+[<delay_in_seconds> | -1] [-r] [<message>]=[<задержка_в_секундах> | -1] [-r] [<сообщение>]
+Shutdown server (-1 cancels a delayed shutdown, -r allows players to reconnect)=Завершение работы сервера (-1 отменяет отложенное завершение работы, -r позволяет игрокам повторно подключиться)
+Server shutting down (operator request).=Завершение работы сервера (по запросу администрации).
+Ban the IP of a player or show the ban list=Заблокировать IP игрока или показать список заблокированных
+The ban list is empty.=Список заблокированных пуст.
+Ban list: @1=Список заблокированных: @1
+You cannot ban players in singleplayer!=Вы не можете забанить игроков в одиночной игре!
+Player is not online.=Игрок не подключен к Сети.
+Failed to ban player.=Не удалось забанить игрока.
+Banned @1.=@1 забанен.
+<name> | <IP_address>=<имя> | <IP_address>
+Remove IP ban belonging to a player/IP=Удалить IP-бан, принадлежащий игроку/IP-адресу
+Failed to unban player/IP.=Не удалось разбанить игрока/IP.
+Unbanned @1.=@1 разбанен.
+<name> [<reason>]=<имя> [<причина>]
+Kick a player=Кикнуть игрока
+Failed to kick player @1.=Не удалось кикнуть @1.
+Kicked @1.=@1 кикнут.
+[full | quick]=[full | quick]
+Clear all objects in world=Очистить ВСЕ объекты во ВСЁМ мире
+Invalid usage, see /help clearobjects.=Недопустимое использование, см. /help clearobjects.
+Clearing all objects. This may take a long time. You may experience a timeout. (by @1)=Очистка всех объектов. Это может занять много времени. У вас может возникнуть тайм-аут. (запущено: @1)
```

Comment:
```suggestion
Clearing all objects. This may take a long time. You may experience a timeout. (by @1)=Удаление всех объектов. Это может занять много времени. Ваш клиент может отключиться из-за тайм-аута. (запущено: @1)
```